corehq/apps/fixtures/templates/fixtures/manage_tables.html

```diff
@@ -71,24 +89,43 @@
             <a data-bind="attr: {href: view_link}" target='_blank'>{% trans "View Table" %}</a>
           </td>
           <td>
+            <!-- ko if: !is_synced() || $parent.unlockLinkedData() -->
             <button class="btn btn-default" data-bind="click: startEditing, visible: saveState() === 'saved'">
-              <i class="fa fa-edit"></i>
-              {% trans "Edit" %}
+              <span>
```

Comment:
why is this `span` here?

Reply:
Did not realize this was extraneous. Removed